client/keys/show.go

```diff
@@ -71,10 +71,16 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 		}
 	} else {
 		pks := make([]cryptotypes.PubKey, len(args))
-		for i, keyref := range args {
-			k, err := fetchKey(clientCtx.Keyring, keyref, clientCtx.AddressCodec)
+		keyFilter := make(map[string]struct{})
+		for i, keyRef := range args {
+			if _, ok := keyFilter[keyRef]; ok {
+				cmd.Printf("WARNING: duplicate keys found: %s.\n", keyRef)
```

Comment:
Why not returning an error here? If we aren't returning an error I think it is mismatched not to just invoke continue

Reply:
Because of my comment: https://github.com/cosmos/cosmos-sdk/pull/18703#pullrequestreview-1777085243. Just in case someone relies on this behavior.